rdbi-recipes-java7/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd really dig it if we could add animal-sniffer to this guy to prevent something I ran into w/ bandolier.

If you have a j8 JDK, you can compile projects with source=1.7 and target=1.7 all day long even if they use java 8 only libraries (like the j8 instant.) it'll just blow up at runtime on j7 jdk's when they can't find the relevant library.

suggest that we configure the maven build plugin to be 1.7, and add the animal-sniffer plugin to, in the verify phase, make sure that nobody used any newfangled java 8 libraries in this project.

see this pr: https://github.com/lithiumtech/bandolier/pull/19/files

(also, we have a plan to get on j8, just waiting on the next stable build of GWT, which should be "soon".)